libraries/core_libs/consensus/include/transaction/gas_pricer.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <boost/circular_buffer.hpp>
+
+#include "final_chain/final_chain.hpp"
+
+namespace taraxa {
+
+class GasPricer {
```

Comment:
maybe a short description of class - what it does, where and how it is used would be nice